File: tests/fixtures/php/comments.php
<?php
/** Coordinates work. */
#[Service]
class Coordinator
{
    /** Current worker. */
    private Worker $worker;

    /** Runs one unit of work. */
    #[Deprecated]
    public function run(): void
    {
        // Keep delegation visible.
        $this->worker->run();
    }
}

File: tests/fixtures/java/comments.java
/** Utilities used by the fixture. */
package comments;

/** Coordinates work. */
class Coordinator {
    /** Current worker. */
    private final Worker worker = createWorker();

    /** Runs one unit of work. */
    @Deprecated
    void run() {
        // Keep delegation visible.
        worker.run();
    }
}
